met-api/src/met_api/services/membership_service.py

```diff
@@ -86,8 +86,8 @@ def _get_membership_details(user_details):
         default_role = CompositeRoles.TEAM_MEMBER.name
         default_membership_type = MembershipType.TEAM_MEMBER
 
-        is_reviewer = CompositeRoles.REVIEWER.value in user_details.get('main_role')
-        is_team_member = CompositeRoles.TEAM_MEMBER.value in user_details.get('main_role')
+        is_reviewer = user_details.get('main_role') == CompositeRoles.REVIEWER.value
+        is_team_member = user_details.get('main_role') == CompositeRoles.TEAM_MEMBER.value
```

Comment:
This was to fix a 500 error - main_role is a single string so it doesn't make much sense to ask if a role is "in" it